upgrading/upgrade-roxctl.adoc

```diff
@@ -1,35 +1,48 @@
 :_content-type: ASSEMBLY
 [id="upgrade-roxctl"]
-= Manually upgrading using the roxctl CLI
+= Manually upgrading using the `roxctl` CLI
 include::modules/common-attributes.adoc[]
 :context: upgrade-roxctl
 
 toc::[]
 
 [role="_abstract"]
-You can upgrade to the latest version of {product-title} from a supported older version.
+You can upgrade to the latest version of {product-title} from a supported older version. For upgrading to {product-title-short} 4.0, you must be using the latest patch release of {product-title-short} 3.74. If you are using an older version, you must first upgrade to {product-title-short} 3.74.
```

Comment:
Can we make this section stand out somehow? This step is critical and we do not want it to be missed.